### DISCUSSION TITLE
Featuring this algo into open source linux repo

### DISCUSSION THREAD

**Discussion by c-a-narain** [pcottle/learnGitBranching#1049] [Ideas]
Why can't we add this as a feature to the Linux repository as visualizing of branches in Linux is somewhat complicated and it makes lots of confusions during working with different branches. By applying this algorithm we can make changes and make ease of working with branches

**Comment by pcottle** [comment]
Ideally that would be great! However it requires a very different environment to run (javascript and a browser) which isn't necessarily suitable for distributing in Linux. Also we only run on data in the browser, not your local repository.

There are other great tools for visualizing your local repository:
https://gitup.co/
https://www.gitkraken.com/git-client

**Reply by c-a-narain** [reply]
Thank you sir I'll follow these
